lib/generators.nix

```diff
@@ -62,10 +63,10 @@ rec {
    * mkKeyValueDefault {} ":" "f:oo" "bar"
    * > "f\:oo:bar"
    */
-  mkKeyValueDefault = {
-    mkValueString ? mkValueStringDefault {}
-  }: sep: k: v:
-    "${libStr.escape [sep] k}${sep}${mkValueString v}";
+  mkKeyValueDefault =
+    { mkValueString ? mkValueStringDefault { }
+    }: sep: k: v:
+      "${libStr.escape [ sep ] k}${sep}${mkValueString v}";
```

Comment:
Should we make newline before `{`?

Reply:
newline before `{` is correct. And the string interpolation doesn't get indented.